src/transxchange/naptan.rs

```diff
@@ -166,23 +168,32 @@ where
         .trim(csv::Trim::All)
         .from_reader(reader);
     let mut stop_points = CollectionWithId::default();
+    let mut stop_areas = CollectionWithId::default();
     for record in reader.deserialize() {
         let stop: NaPTANStop =
             record.with_context(|_| "Error parsing the CSV record into a Stop")?;
         if stop.status != "act" {
             continue;
         }
-        let stop_area_id = stops_in_area.get(&stop.atco_code).cloned().ok_or_else(|| {
-            format_err!(
-                "Failed to find the corresponding StopArea for the StopPoint '{}'",
-                stop.atco_code
-            )
-        });
-        let stop_area_id = skip_fail!(stop_area_id);
         let coord = Coord {
             lon: stop.longitude,
             lat: stop.latitude,
         };
+        let stop_area_id = match stops_in_area.get(&stop.atco_code).cloned() {
+            Some(stop_area_id) => stop_area_id,
+            None => {
+                // If the stop point don't have a corresponding stop area
+                // create the stop area based on stop point information
+                let id = format!("SA:{}", stop.atco_code);
```

Comment:
we can use the same prefix like https://github.com/CanalTP/transit_model/blob/8f7f68b7ec65823278c571d1f6556bdbfbfc7c6f/src/ntfs/read.rs#L94

Reply:
Yep, fixed.